behave/features/steps/fala_end_to_end_tests.py

```diff
@@ -10,15 +10,25 @@ def step_impl_homepage(context):
     title_xpath = context.helperfunc.find_by_xpath(
         "//html/body/div/main/div/div/h1"
     ).text.replace("\n", " ")
-    assert title_xpath, f"{FALA_HEADER}"
+    assert title_xpath == FALA_HEADER
 
 
 @step('I provide the "{location}" details')
 def step_impl_input_location(context, location):
     input_id = context.helperfunc.find_by_id("id_postcode")
     input_id.send_keys(location)
     input_xpath = context.helperfunc.find_by_xpath('//input[@id="id_postcode"]')
-    assert input_xpath.get_attribute("value"), location
+    input_value = input_xpath.get_attribute("value")
+    assert input_value == location
+
+
+@step('I provide an organisation name "{organisation}"')
+def step_impl_input_organisation(context, organisation):
+    input_id = context.helperfunc.find_by_id("id_name")
+    input_id.send_keys(organisation)
+    input_xpath = context.helperfunc.find_by_xpath('//input[@id="id_name"]')
```

Comment:
You've already have a variable called `input_id` which has found the id "id_name" so you probably don't need this xpath anymore.

Reply:
good spot, amended this pattern in this step + location step